src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(Lottery.getRandomBall());
+        System.out.println(Lottery.getRandomBall());
+        System.out.println(Lottery.getRandomBall());
```

Comment:
Don't use static methods in your solution. Instead of calling static methods on the Lottery class, create an instance of the Lottery class and call the method on that instance. This will make your code more object-oriented and easier to maintain.